src/xrpld/app/tx/detail/Payment.cpp

```diff
@@ -547,7 +567,7 @@ Payment::doApply()
 
     // The source account does have enough money.  Make sure the
     // source account has authority to deposit to the destination.
-    if (reqDepositAuth)
+    if (depositAuth)
```

Comment:
Shouldn't this be reqDepositAuth?

Reply:
No, cause `reqDepositAuth`  divided intro 2 check, and one of them here - `depositAuth`, and other one inside `verifyDepositPreauth`, because it common for all other transactions